index.js

```diff
@@ -1,88 +1,125 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+var counter = 1
```

Comment:
This works, but the point here is to demonstrate that we can use `let` to declare a variable and then we're able to reassign it. So you could do:

```javascript
let counter = 1
counter = 2
```

And this way we're not reassigning the variable, just changing its value. It may seem small and trivial in this example, but best to get in the habit as you'll see its use down the line!